src/pages/AdminProfilePage/AdminProfilePage.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { ChakraProvider, Button, useDisclosure, Input, Image } from '@chakra-ui/react';
+
+import styles from './AdminProfilePage.module.css';
+import DeleteAccountModal from '../../components/AdminProfilePage/DeleteAccountModal/DeleteAccountModal';
+import SendLinkModal from '../../components/AdminProfilePage/SendLinkModal/SendLinkModal';
+import FYALogoLarge from '../../assets/fya-logo-large.svg';
+import PenImage from '../../assets/Vector.png';
+
+// TODO:
+// - Remove ChakraProvider once provider is moved to index.js
+
+const AdminProfilePage = () => {
+  const {
+    isOpen: isOpenDeleteModal,
+    onOpen: onOpenDeleteModal,
+    onClose: onCloseDeleteModal,
+  } = useDisclosure();
+
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  return (
+    <ChakraProvider>
+      <div className={styles['sub-header']}>
+        <Button
+          onClick={onOpen}
+          color="white"
+          className={styles['share-registration-box']}
+          borderRadius="15px"
+          bg="#1F2F38"
+          fontSize="20px"
+          size="md"
+          width="260px"
+        >
+          Share Registration Link
+        </Button>
+        <SendLinkModal isOpen={isOpen} onClose={onClose} />
+
+        <div className={styles['welcome-header']} colorScheme="blue">
+          <h1>WELCOME!</h1>
+        </div>
+
+        <div className={styles['logout-button']}>
+          <Button
+            onClick={null}
+            colorScheme="teal"
+            borderRadius="15px"
+            bg="#345E80"
+            size="lg"
+            width="170px"
+            fontSize="27px"
+          >
+            Logout
+          </Button>
+        </div>
+      </div>
+
+      <div className={styles['fya-logo-large']}>
+        <img boxSize="100px" src={FYALogoLarge} alt="Find Your Anchor Logo" />
+      </div>
+
+      <div className={styles['background-box']}>
+        <div className={styles['profile-name']}>
+          <div className={styles['first-name-container']}>
+            <h1>First Name:</h1>
+            <div className={styles['name-input-container']}>
+              <Input
```

Comment:
Please check with the designers on this, but I believe the first and last name fields should only be editable when the user clicks on the pencil icon. Can we add this functionality here if that is what the designers agree with?

Reply:
Confirmed with the designers that this is the intended functionality. The user should click on the pencil icon for a field in order to edit it and then would click on a checkmark icon to save their changes. https://ctcuci.slack.com/archives/C02JRBG5W6R/p1645497669133759?thread_ts=1645423847.909649&cid=C02JRBG5W6R

Reply:
Resolved.